src/lib/api.ts

```
@@ -1,5 +1,41 @@
+  if (hostname.includes("replit")) return "https://ecc-triage-node-staging.azurewebsites.net";
+
+  return "";
+}
+
+function joinUrl(base: string, path: string): string {
+  if (!base) return path;
+  if (/^https?:\/\//i.test(path)) return path;
+  return `${base}${path.startsWith("/") ? "" : "/"}${path}`;
+}
+
+const API_PATH_PREFIX = import.meta.env.VITE_API_BASE ?? "/api";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'API_PATH_PREFIX'. Consider to delete it or rename it.

'API_PATH_PREFIX' is defined here:


(lint/suspicious/noRedeclare)